### DISCUSSION TITLE
share my answer for assignment of Episode 4

### DISCUSSION THREAD

**Discussion by hbsun2113** [mlc-ai/mlc-en#34] [General]
Please let me know if anything wrong
https://colab.research.google.com/gist/hbsun2113/1329eba39ab1dda46015f9c5244f9d2d/-assignment1-ipynb.ipynb

**Comment by Hzfengsy** [comment]
Thanks @hbsun2113 for sharing your answers. 

Putting every compute in one `PrimFunc` may cause problems during further optimizations. An alternate way is using `call_tir` multiple times and only computing a single PyTorch operator one time (e.g. `emit_te(topi.nn.conv2d, ...)`)
